# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  e83a29afbd4b73d1049c0317b958e336b80b70ce
Feature: Bug 1831667: Re-establish total and unread/selected counts, take 2

diff --git a/mail/base/content/about3Pane.js b/mail/base/content/about3Pane.js
--- a/mail/base/content/about3Pane.js
+++ b/mail/base/content/about3Pane.js
@@ -4151,17 +4151,18 @@ var threadPaneHeader = {
   },
 
   updateStatusBar() {
     if (!this.countsElement) {
       return;
     }
     // Not updating if we're not viewing the current folder.
     if (
-      top.window.document.getElementById("tabmail").currentAbout3Pane.gFolder !=
+      !gFolder ||
+      top.window.document.getElementById("tabmail")?.currentAbout3Pane?.gFolder !=
       gFolder
     ) {
       return;
     }
     let count = gFolder?.getTotalMessages(false) || gDBView?.rowCount;
     if (count === undefined) {
       // Too early to show, let's get out of here.
       return;
